op-node/rollup/superchain.go

```diff
@@ -0,0 +1,96 @@
+package rollup
```

Comment:
Why is this in the rollup package & not chain-config?

Reply:
Either place would work. I think I can move it. It's one of the primary places to instantiate rollup configs, so hence the original choice to put it next to the rollup config type.